src/main/java/net/mcreator/ui/minecraft/states/IPropertyData.java

```diff
@@ -136,6 +137,7 @@ public IPropertyData<?> deserialize(JsonElement json, Type typeOfT, JsonDeserial
 				throws JsonParseException {
 			JsonObject jsonObject = json.getAsJsonObject();
 			PropertyData<?> data = gson.fromJson(jsonObject, typeMappings.get(jsonObject.get("type").getAsString()));
+			data.uuid = UUID.randomUUID(); // init UUID for deserialized property

```

Comment:
This should not be needed as default constructor () is always called

Reply:
The same is done for `GUIComponent`s.

Reply:
Ok, probably not needed though